_posts/2018-06-21-vision-for-rust-and-wasm.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Our Vision for Rust and WebAssembly
+---
+
+<meta charset="utf-8" />
+
+**Surgically inserting Rust compiled to WebAssembly should be the best choice
+for speeding up the most performance-sensitive JavaScript code paths.** Do *not*
+throw away your existing code base, because Rust [plays well with
+others][better-for-all]. Regardless if you are a Rust or Web developer, your
+natural workflow shouldn't change because Rust compiled to wasm integrates
+seamlessly into your preferred tools.
+
+This blog post will expand on these aspirations and describe where we stand in
+relation to them right now. In a series of follow up posts, we will talk about
+the next steps for each major component of the Rust and WebAssembly ecosystem.
+
+<style>
+hr {
+    margin-top: 2em;
+    margin-bottom: 2em;
+}
+</style>
+
+--------------------------------------------------------------------------------
+
+> [Are you interested in helping us make these ideals into reality? Join the Rust
+and WebAssembly domain working group!][get-involved]
+
+--------------------------------------------------------------------------------
+
+## Why Focus on Performance-Sensitive Code?
```

Comment:
Is it supposed to be Performance-Sensitive and not Performance Sensitive?

Reply:
I actually only learned this rule like a month or two ago :-p

http://blog.esllibrary.com/2013/01/10/when-to-use-hyphens-rules-for-multiple-word-adjectives/

> If the multiple-word adjective comes before a noun, use hyphens.
>
> Examples:
>
> * She gave me an up-to-date report.
> * We used computer-generated images in our presentation.
> * After the reading exercise, answer these follow-up questions.